.github/workflows/testing.yml

```diff
@@ -1,6 +1,8 @@
 name: Run Test Suite
 
-on: [push]
+on:
+  - push
+  - pull_request
```

Comment:
likewise with linting, I'm not certain what types of pull_request event should be specified here.